src/xdocs/config_naming.xml

```diff
@@ -163,14 +163,27 @@
           no abbreviations (enforce camel case phrase) and
           allow no abbreviations to use (camel case phrase) and allow XML and URL abbreviations.
         </p>
+        <p>Configuration:</p>
         <source>
 &lt;module name="AbbreviationAsWordInName"&gt;
   &lt;property name="tokens" value="VARIABLE_DEF,CLASS_DEF"/&gt;
   &lt;property name="ignoreStatic" value="false"/&gt;
-  &lt;property name="allowedAbbreviationLength" value="1"/&gt;
+  &lt;property name="allowedAbbreviationLength" value="0"/&gt;
```

Comment:
Setting the value to 0 to match the description above (camel case phrase).

Reply:
good catch but please make one more example with length=1 or 2 or 3 to show user what is possible to archive by other values.
In one example please make "xxxStatic=true", please make example with validation over only variables not a class names. Make example for user specific abbreviation like `allowedAbbreviations="CSV"` and line that show that XML become illegal.